doc/sphinx-guides/source/installation/advanced.rst

```diff
@@ -0,0 +1,30 @@
+=====================
+Advanced Installation
+=====================
+
+Advanced installations are not officially supported but here we are at least documenting some tips and tricks that you might find helpful. You can find a diagram of an advanced installation in the :doc:`prep` section.
+
+.. contents:: |toctitle|
+	:local:
+
+Multiple Glassfish Servers
+--------------------------
+
+The main thing to know about running multiple Glassfish servers is that only one can be the dedicated timer server, as explained in the :doc:`/admin/timers` section of the Admin Guide.
+
+Detecting Which Glassfish Server a User Is On
++++++++++++++++++++++++++++++++++++++++++++++
+
+If you have successfully installed multiple Glassfish servers behind a load balancer you might like to know which server a user has landed on. A straightforward solution is to place a file called ``host.txt`` in a directory that is served up by Apache such as ``/var/www/html`` and then configure Apache not to proxy requests to ``/host.txt`` to Glassfish. Here are some example commands on RHEL/CentOS 7 that accomplish this::
```

Comment:
Is "a directory that is served up by Apache" enough detail?  It is for me; but I'm not sure if that's true of all users (aka - somebody who doesn't realize they need `DocumentRoot` and mod_alias enabled)?

Otherwise, looks good to me.

Reply:
I think it's enough detail. For years RHEL/CentOS has served up files in `/var/www/html` out of the box with no special config. Thanks for taking a look.